docs/Toy/Toy_Ch1.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>You could add, that quite a bit of this text is taken from the website</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added some more text about this being the case